skills/evaluation/scripts/evaluator.py

```
@@ -277,198 +336,292 @@ def filter(self, **criteria) -> List[Dict]:
+
+    # 2. Build a standard test set
+    test_set = TestSet("demo").create_standard_tests()
+    print(f"Test set: {test_set.name}")
+    print(f"Test count: {len(test_set.tests)}")
+    print(f"Complexity distribution: {test_set.get_complexity_distribution()}\n")
+
+    # 3. Run evaluation
+    runner = EvaluationRunner(evaluator, test_set)
+    summary = runner.run_all(verbose=True)
+
+    print(f"\n--- Summary ---")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)